server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -6,6 +6,8 @@ services:
   cms:
     image: taccwma/core-cms:latest
     volumes:
+      - ../cms/settings_custom.py:/code/taccsite_cms/settings_custom.py
+      - ../cms/settings_local.py:/code/taccsite_cms/settings_local.py
```

Comment:
Causes bug. Bug fixed by removing this (https://github.com/TACC/Core-Portal/pull/637).